frontend/src/views/WheresMyGene/common/store/reducer.ts

```diff
@@ -418,10 +421,11 @@ function setXAxisHeight(state: State, action: PayloadAction<number>): State {
 
 function setFilteredCellTypes(
   state: State,
-  action: PayloadAction<State["filteredCellTypes"]>
+  action: PayloadAction<SetFilteredCellTypesPayload>
```

Comment:
wanted to use the same reducer/action so that we're updating the cell type names and ids at the same time